app/src/main/java/com/boardgamegeek/ui/GameCreditsFragment.kt

```diff
@@ -84,7 +80,7 @@ class GameCreditsFragment : Fragment() {
     private fun colorize(@ColorInt iconColor: Int) {
         if (!isAdded) return
 
-        listOf(game_info_designers, game_info_artists, game_info_publishers, game_info_categories, game_info_mechanics, game_info_expansions, game_info_base_games)
+        listOf(game_info_designers, game_info_artists, game_info_publishers, game_info_categories, game_info_mechanics)
```

Comment:
Move this same call for the new rows into the `GameFragment`. The code should already exist - just need to add the new view groups.

Reply:
I don't understand what do you mean. I guess you are referring to this [part](https://github.com/ccomeaux/boardgamegeek4android/pull/124/files#diff-53926588443e39a8c6f3d339a54abfa3R125-R126). Is something missing there?